test/tests/meshgenerators/sidesets_from_points_generator/sidesets_ambiguity.i

```diff
@@ -0,0 +1,350 @@
+[GlobalParams]
+  prevent_boundary_ids_overlap = false
```

Comment:
I thought this had to be `true` to run into the issue?

Reply:
Otherwise how does the `preparedness` test work? Or did you modify the mesh generators? I guess I assumed this was a copy of that test

Reply:
Ah, shoot, since this is a copy of the preparedness test instead of a modification of it, you don't see the specific diffs highlighted, do you?  There's just one earlier but it's a big one:

```
      stitch_boundaries_pairs = '1   3;
                                 1   3;
                                 0   0' # 0 0 will leave a slit mesh
```

I'm not relying on auto-renumbering to tear the mesh here; I'm now deliberately leaving one of the input meshes unstitched.

Reply:
got ya 👍  I see that in there now